Library/Homebrew/cmd/doctor.rb

```diff
@@ -92,6 +92,26 @@ def check_for_macgpg2
     end
   end
 
+  # Anaconda installs multiple system & brew dupes, including OpenSSL, Python,
+  # sqlite, libpng, Qt, etc. Regularly breaks compile on Vim, MacVim and others.
+  # Is flagged as part of the *-config script checks below, but people seem
+  # to ignore those as warnings rather than extremely likely breakage.
+  def check_for_anaconda
+    return unless which("anaconda")
+    return unless which("conda-server")
+
+    if which("anaconda") then <<-EOS.undent
```

Comment:
@mikemcquaid Not sure if I'm misunderstanding you.

Are you saying we shouldn't warn if `anaconda` is in someone's `$PATH` but Python/cURL/etc still point to either us or the system, or are you saying we should also check one of the dupes for prefix in case people try to remove the specific binaries we check from the `$PATH` as a doctor hack?